user/service.go

```diff
@@ -0,0 +1,24 @@
+package user
+
+import (
+	"context"
+	"time"
+)
+
+type UserEntity struct {
+	dataSource UserDataSource
+}
+
+// Set time.Now outside of method for testing
+var timeNow = time.Now
+
+func (u UserEntity) CreateNewUser(ctx context.Context) (string, error) {
+	preferredTime := PreferredTime{Hour: timeNow().UTC().Hour(), Min: timeNow().UTC().Minute()}
```

Comment:
I would call `time.Now` first to create an instance of time and then get the Hour and Min.

Reply:
Done.